### PR TITLE
feat/async-popup

### PR DESCRIPTION
**Pull request by JeffersonCarvalh0**

### CODE REVIEW

File: packages/async-popup/.gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.DS_Store
+.merlin
+.bsb.lock
+npm-debug.log
```
Comment (fakenickels) [inline packages/async-popup/.gitignore]: let's add the .bs.js to ignore too?

**Comment by fakenickels** [comment]
Just missing a README!

File: packages/async-popup/.gitignore (hunk 1 above)
Comment (JeffersonCarvalh0) [reply packages/async-popup/.gitignore]: Alright. I'll add it to the template too

Comment (fakenickels) [reply packages/async-popup/.gitignore]: thx

**Pull request merged**